.github/scripts/README.md

```diff
@@ -56,3 +56,9 @@ Exceptions to AWS for self hosted:
 New runner types can be added by committing changes to `.github/scale-config.yml`. Example: https://github.com/pytorch/pytorch/pull/70474
 
 > NOTE: New runner types can only be used once the changes to `.github/scale-config.yml` have made their way into the default branch
+
+### Testing [pytorch/builder](https://github.com/pytorch/builder) changes
+
+In order to test changes to the builder scripts, submit a fake PR against PyTorch that specifies your PR's branch as `builder_branch` in [`.github/templates/common.yml.j2`](https://github.com/pytorch/pytorch/blob/master/.github/templates/common.yml.j2#LL10C9-L10C23). Make sure to regenerate workflow files with `.github/regenerate.sh` (see above).
```

Comment:
Hmm, this link would become invalid really soon, should we have an absolute link here (i.e. https://github.com/pytorch/pytorch/blob/26beb46da4b4c778e8162a67711ebe8fcd1408ae/.github/templates/common.yml.j2#LL10C9-L10C23 )
```suggestion
In order to test changes to the builder scripts, submit a fake PR against PyTorch that specifies your PR's branch as `builder_branch` in [`.github/templates/common.yml.j2`](https://github.com/pytorch/pytorch/blob/26beb46da4b4c778e8162a67711ebe8fcd1408ae/.github/templates/common.yml.j2#LL10C9-L10C23). Make sure to regenerate workflow files with `.github/regenerate.sh` (see above).
```